tests/acceptance/features/apiSpaces/notification.feature

```diff
@@ -21,108 +21,159 @@ Feature: Notification
-        "datetime",
-        "message",
-        "messageRich",
-        "messageRichParameters",
-        "notification_id",
-        "object_id",
-        "object_type",
-        "subject",
-        "subjectRich",
-        "user"
-      ],
-      "properties": {
-        "app": {
-          "type": "string",
-          "enum": ["userlog"]
-        },
-        "message": {
-          "type": "string",
-          "enum": ["Alice Hansen added you to Space notification checking"]
-        },
-        "messageRich": {
-          "type": "string",
-          "enum": ["{user} added you to Space {space}"]
-        },
-        "messageRichParameters": {
-          "type": "object",
-          "required": [
-            "space",
-            "user"
-          ],
-          "properties": {
-            "space": {
-              "type": "object",
-              "required": [
-                "id",
-                "name"
-              ],
-              "properties": {
-                "id": {
-                  "type": "string",
-                  "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}\\$[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}![a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
-                },
-                "name": {
-                  "type": "string",
-                  "enum": ["notification checking"]
+      """
+      {
+        "type": "object",
+        "required": [
+          "app",
+          "datetime",
+          "message",
+          "messageRich",
+          "messageRichParameters",
+          "notification_id",
+          "object_id",
+          "object_type",
+          "subject",
+          "subjectRich",
+          "user"
+        ],
+        "properties": {
+          "app": {
+            "type": "string",
+            "enum": [
+              "userlog"
+            ]
+          },
+          "message": {
+            "type": "string",
+            "enum": [
+              "Alice Hansen added you to Space notification checking"
+            ]
+          },
+          "messageRich": {
+            "type": "string",
+            "enum": [
+              "{user} added you to Space {space}"
+            ]
+          },
+          "messageRichParameters": {
+            "type": "object",
+            "required": [
+              "space",
+              "user"
+            ],
+            "properties": {
+              "space": {
+                "type": "object",
+                "required": [
+                  "id",
+                  "name"
+                ],
+                "properties": {
+                  "id": {
+                    "type": "string",
+                    "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}\\$[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}![a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
may be we can make a function and name what type of regex pattern is ths and substitute. may be we can use such pattern in other place as well.

Reply:
I wanted to check specific message value.

Reply:
It's space id like `3db75261-afa8-4224-9c10-f8459fd047d7$1eb6a3e9-5dba-4887-b1b2-0c26d73d4e83!1eb6a3e9-5dba-4887-b1b2-0c26d73d4e83`

We have pattern for checking spaceID but my spaceUUID contains `!` . self::getUUIDv4Regex() -> maybe it is bug(because objectId is normal spaceId). 

<img width="1359" alt="Screenshot 2023-05-16 at 21 57 37" src="https://github.com/owncloud/ocis/assets/84779829/ff7199ac-7931-49ba-9f9a-69d5150c0eac">


```
public static function getSpaceIdRegex(): string {
		return self::getUUIDv4Regex() . '\\\$' . self::getUUIDv4Regex();
```
Otherwise I have to add  something like:

```
public static function getFullSpaceIdRegex(): string {
		return self::getUUIDv4Regex() . '\\\$' . self::getUUIDv4Regex(). '\\\!' . self::getUUIDv4Regex();
```